lib/internal/url.js

```diff
@@ -1289,6 +1291,7 @@ function getPathFromURLWin32(url) {
       }
     }
   }
+  pathname = pathname.replace(forwardSlashRegEx, '\\');
```

Comment:
If you think it makes anything more readable here or elsewhere, you can use `path.win32.sep` and/or `path.posix.sep` and/or `path.sep`. (No strong opinion from me on whether or not they should be used here. Just pointing 'em out in case you didn't know about 'em.)

Reply:
I'm not sure I have a strong opinion either.